Makefile.vars.mk

```diff
@@ -1,6 +1,5 @@
 # Current Operator version
-VERSION ?= $(shell date +%Y-%m-%d_%H-%M-%S)
-VERSION := $(VERSION)
+VERSION ?=
 
 # Default bundle image tag
 BUNDLE_IMG ?= controller-bundle:$(VERSION)
```

Comment:
Since I've been looking into the `$(VERSION)` variable, I've come to believe that the `BUNDLE_IMG` variable is not used anymore.